Modules/Scripted/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(modules
   CropVolumeSequence
   DataProbe
   ImportItkSnapLabel
+  LineProfile
```

Comment:
This module includes a usage of `scipy`, so similar to Endoscopy module being added below, LineProfile should be specified only if Slicer_USE_SCIPY option is ON.